guide/14-deep-learning/panoptic_segmentation_with_maxdeeplab.ipynb

```diff
@@ -0,0 +1,189 @@
+{
```

Comment:
<p>put model name in backticks <code>MaXDeepLab</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1431/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Putting the model name in backticks shouldn't be necessary here as we are not saying anything about the code or the API.